PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/build.gradle

```diff
@@ -10,5 +10,7 @@ dependencies {
     implementation 'com.microsoft.azure:azure-toolkit-common-lib'
     implementation 'com.microsoft.azure:azure-toolkit-ide-common-lib'
     aspect 'com.microsoft.azure:azure-toolkit-common-lib'
-
-}
+    implementation group: 'org.json', name: 'json', version: '20240303'
+    implementation 'com.microsoft.azure:applicationinsights-core:2.6.2'
+    implementation 'com.microsoft.azure:applicationinsights-web:2.6.2'
```

Comment:
Should we use specific versions in this Gradle file? Looking at lines 10-12 makes me think there is another place in the repository we control library versions.

Reply:
will work on this in the telemtry PR (I've separated them)

Reply:
I also checked for instances that the project builds fine without defining specific versions and took the version numbers out completely. Others like org.junit.jupiter and org.json don't have successfull builds when you don't state the version.